### ISSUE TITLE
Editor crashes when my app was updated to latest xamarin.forms 3.4 .0.1008975 

### ISSUE THREAD

**Issue by kylehuang818a** [xamarin/Xamarin.Forms#4832]
### Description

My App crashes by using Editor, when I updated xamarin.forms from 3.2 to3.4 .

It crashes only when  my smartphone's orientation is landscape.  

It happens only by Andriod. There is no such a  bug by iOS.

### Steps to Reproduce

1. Andriod, uses portable project, one of tagets is .NET Framework 4.5
2. smartphone's orientation is landscape ( portrait is ok)
3. Input some text for Editor then press "DONE"
4. Crashes:

Java.Lang.IllegalStateException: focus search returned a view that wasn't able to take focus!

### Basic Information

- Version with issue:
- Last known good version:
- IDE: 
Visual Studio 2017
- Platform Target Frameworks:  
   
  - Android: <  7.1 -->

**Comment by kylehuang818a** [comment]
Affected device: Moto G(5S)

**Comment by kingces95** [comment]
Please attached a zipped reproduction. Ideally reduced (start with a blank template project).

**Comment by byishimoaudace** [comment]
The issue still exists in Xamarin.Forms 3.4.0.1029999

**Comment by Brosten** [comment]
Very easy to reproduce. New Xamarin Forms Project, 
XF version 3.5
Setup page like this:
```
    <StackLayout>
      <Editor/>
      <Button Text="SomeText" IsEnabled="False"/>
   </StackLayout>

```
Run on Android phone.
Put it in landscape
Enter edit mode of the editor
Push "Done" button.

Tested with Samsung Galaxy S7 running Android 8.1. Google keyboard.
@samhouts

**Comment by pauldipietro** [comment]
Those steps cause the aforementioned crash on an API 27 emulator.

**Comment by Brosten** [comment]
A lazy workaround I'm using until fixed in XF
```
   <StackLayout>
      <Editor/>
      <Button Text="SomeText" IsEnabled="False"/>
      <Editor x:Name="CrashStopperHack" 
                    HeightRequest="0" 
                    WidthRequest="0"
                    Focused="CrashStopperHack_Focused"/>
   </StackLayout>
```

And in code behind
```
      private void CrashStopperHack_Focused(object sender, FocusEventArgs e)
      {
         CrashStopperHack.Unfocus();
      }
```

**Comment by andreinitescu** [comment]
Should be fixed by #5320
